src/idsk/components/accordion/template.njk

```diff
@@ -0,0 +1,32 @@
+{% set id = params.id %}
+{% set headingLevel = params.headingLevel if params.headingLevel else 2 %}
+
+<div class="govuk-accordion {%- if params.classes %} {{ params.classes }}{% endif -%}" data-module="idsk-accordion" id="{{ id }}"
+{%- for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
+<div class="govuk-accordion__controls">
+  <button class="govuk-accordion__open-all" id="accordionButton" data-open-title="{{ params.openTitle if params.openTitle else 'Otvoriť všetko' }}"
+  data-close-title="{{ params.closeTitle if params.closeTitle else 'Zatvoriť všetko' }}" type="button" aria-expanded="false">
+     <span class="govuk-visually-hidden">sections</span></button>
```

Comment:
sections by som tiez parametrizoval.. citacka to v tomto pripade by precitala: "Otvorit vsetko sections" co nedava zmysel velmi.. dajme to tak isto ako ten button akurat  tu bude default " - sekcie". Pozor lebo bude ot treba upravit aj v js